ui/src/app/services/workflow/workflow.service.ts

```diff
@@ -167,6 +168,13 @@ export class WorkflowService {
     );
   }
 
+  getJobDynamicFormParts(): Observable<DynamicFormPart[]> {
```

Comment:
Why does this method need to return an `Observable`?

Reply:
This in the future will be an API backend call. It will be easier to refactor.